rescheduler/rescheduler.go

```diff
@@ -112,6 +123,11 @@ func main() {
 				if len(criticalPods) > 0 {
 					for _, pod := range criticalPods {
 						glog.Infof("Critical pod %s is unschedulable. Trying to find a spot for it.", podId(pod))
+						k8sApp := "unknown"
+						if l, found := pod.ObjectMeta.Labels["k8s-app"]; found {
```

Comment:
@brancz have you opened issue to introduce standard/best practices for labelling k8s-apps?